### DISCUSSION TITLE
IEX Cloud Cache vs Database vs Redis

### DISCUSSION THREAD

**Discussion by chronark** [perfolio/perfolio#150] [Feature Requests]
Currently we can not use the IEX cache because it's only available in the individual and enterprise plan (enterprise is $200/m)

Right now we are caching everything in FaunaDB which works but can potentially be more expensive than other solutions.

Alternative 1: Upstash (Serverless Redis)
https://upstash.com/#section-pricing

**Comment by webersni** [comment]
Love it!

**Comment by chronark** [comment]
We're using redis for companies and prices now. seems to work great.
